egs2/TEMPLATE/asr1/pyscripts/utils/extract_xvectors.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+#  2022, Hitachi LTD.; Nelson Yalta
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+import argparse
+import kaldiio
+import logging
+from pathlib import Path
+import sys
+import torch
+import os
+import numpy as np
+
+from tqdm.contrib import tqdm
+
+from espnet2.fileio.sound_scp import SoundScpReader
+
+
+def get_parser():
+    """Construct the parser."""
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser.add_argument(
+        "--pretrained",
```

Comment:
I want to make it consistent in naming.
```suggestion
        "--pretrained_model",
```